src/azure/Templates/AzurePagingMethodTemplate.cshtml

```diff
@@ -6,7 +6,15 @@
 @inherits AutoRest.Python.PythonTemplate<AutoRest.Python.Azure.Model.MethodPya>
 @{
 string methodName = Model.Name.ToPythonCase();
+string trace_decorator = "";
+string sync_trace_decorator = "";
 }
+@if (Model.AddTracingDecorators)
+{
+trace_decorator = AsyncMode? "@distributed_trace_async" : "@distributed_trace";
+sync_trace_decorator = "@distributed_trace";
+}
+    @(sync_trace_decorator)
```

Comment:
I would not do a variable, and do directly the "if Model.AddTracingDecorators" to avoid an empty line if --trace is not used

Reply:
Interestingly this does not actually create a new line.